### ISSUE TITLE
[Word Desktop]: NVDA is not announcing the list position as well as list level (nesting).

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8279]
### Steps to reproduce:
1. Open Sample doc
2. Use Down arrow and navigate to some list
3. Observe that NVDA is not announcing list type of most of the bullets. (Square, Dash-)
4. Also observed that there is no announcement of list position.
5. Observe that there is no announcement of list level (Nesting)

 ### Expected behavior:
NVDA should announce the list position and list level (nesting).

### Actual behavior:
NVDA is not announcing the list position as well as list level (nesting).

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by Adriani90** [comment]
@WiproPSAccTest if word Desktop is the regular MS Word Version, there are already issues to These Problems. I am closing this one in favor of #8666, #7742, #7804  and #6815.
